crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -758,6 +761,192 @@ impl RunArgs {
+        }
+
+        if self.since.is_some() {
+            telemetry.track_arg_usage("since", true);
+        }
+
+        if self.include_dependencies {
+            telemetry.track_arg_usage("include-dependencies", true);
+        }
+
+        if self.no_deps {
+            telemetry.track_arg_usage("no-deps", true);
+        }
+
+        if self.no_cache {
+            telemetry.track_arg_usage("no-cache", true);
+        }
+
+        if self.daemon {
+            telemetry.track_arg_usage("daemon", true);
+        }
+
+        if self.no_daemon {
+            telemetry.track_arg_usage("no_daemon", true);
+        }
+
+        if self.only {
+            telemetry.track_arg_usage("only", true);
+        }
+
+        if self.parallel {
+            telemetry.track_arg_usage("parallel", true);
+        }
+
+        if self.pkg_inference_root.is_some() {
+            telemetry.track_arg_usage("pkg-inference-root", true);
+        }
+
+        if self.profile.is_some() {
+            telemetry.track_arg_usage("profile", true);
+        }
+
+        if self.anon_profile.is_some() {
+            telemetry.track_arg_usage("anon-profile", true);
+        }
+
+        if self.remote_only {
+            telemetry.track_arg_usage("remote-only", true);
+        }
+
+        if self.remote_cache_read_only {
+            telemetry.track_arg_usage("remote-cache-read-only", true);
+        }
+
+        if self.summarize.is_some() {
+            telemetry.track_arg_usage("summarize", true);
+        }
+
+        if self.experimental_space_id.is_some() {
+            telemetry.track_arg_usage("experimental-space-id", true);
+        }
+
+        // track values
+        if let Some(dry_run) = &self.dry_run {
+            telemetry.track_arg_value(
+                "dry-run",
+                match dry_run {
+                    DryRunMode::Text => "text",
+                    DryRunMode::Json => "json",
+                },
+                EventType::NonSensitive,
+            );
+        }
+
+        if self.cache_workers != DEFAULT_NUM_WORKERS {
+            telemetry.track_arg_value("cache-workers", self.cache_workers, EventType::NonSensitive);
+        }
+
+        if let Some(concurrency) = &self.concurrency {
+            telemetry.track_arg_value("concurrency", concurrency, EventType::NonSensitive);
+        }
+
+        if !self.global_deps.is_empty() {
+            telemetry.track_arg_value("global-deps", self.cache_workers, EventType::NonSensitive);
+        }
+
+        if let Some(graph) = &self.graph {
+            telemetry.track_arg_value("graph", graph, EventType::NonSensitive);
+        }
+
+        if self.env_mode != EnvMode::default() {
+            telemetry.track_arg_value(
+                "env-mode",
+                match self.env_mode {
+                    EnvMode::Infer => "infer",
+                    EnvMode::Loose => "loose",
+                    EnvMode::Strict => "strict",
+                },
```

Comment:
We should implement `Display` on `EnvMode` and consolidate the logic there (which in turn should probably make use of the `Serialize` implementation as I think we want to keep those behaviors in sync)